pkg/types/types.go

```diff
@@ -66,6 +66,9 @@ type NetConf struct {
 	IPAM         struct {
 		Type string `json:"type,omitempty"`
 	} `json:"ipam,omitempty"`
+	IPAM6 struct {
+		Type string `json:"type,omitempty"`
+	} `json:"ipam6,omitempty"`
```

Comment:
I agree, it would be better to have one ipam config block that contains a list of addresses. The changes that I've proposed are based on PR #279 (see this line: https://github.com/containernetworking/cni/pull/279/files#diff-b257c660535eb4a7279ff8eb1cc07214R46). PR #279 needs to be modified to reflect this, if I'm not mistaken. Having a list of addresses would help for multiple IPv6 addresses, e.g. when we want to display both an LLA and another address (ULA or GUA) for a pod interface.